docs/extra_pages.dox

```diff
@@ -115,6 +115,146 @@
  * import polycap
  * \endcode
  *
+ * \page pyth_docs Python Docs
+ * 
+ * \section Profile Profile
+ * Class containing information about a polycapillary profile shape.
+ * When this class is no longer required, it is the user's responsability to free the memory using ``Profile.__dealloc__``.
```

Comment:
Dit is incorrect: de gebruiker moet nooit enige `__dealloc__` methods oproepen. Python doet dit automatisch voor u wanneer het object zijn refcount naar 0 gaat en vernietigd wordt.

In Python moet je vrijwel nooit methods callen die omringt zijn door dubbele underscores.